### PR TITLE
Added checks so local player isn't added to nearby clan members.

### PR DESCRIPTION
**Pull request by DarkSlaayer**
Currently local player is added to nearby clan members.

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/clanchat/ClanChatPlugin.java [hunk 1]
```diff
@@ -72,14 +42,18 @@
...
 import net.runelite.client.plugins.PluginDescriptor;
-import static net.runelite.client.ui.JagexColors.CHAT_CLAN_NAME_OPAQUE_BACKGROUND;
-import static net.runelite.client.ui.JagexColors.CHAT_CLAN_NAME_TRANSPARENT_BACKGROUND;
-import static net.runelite.client.ui.JagexColors.CHAT_CLAN_TEXT_OPAQUE_BACKGROUND;
-import static net.runelite.client.ui.JagexColors.CHAT_CLAN_TEXT_TRANSPARENT_BACKGROUND;
 import net.runelite.client.ui.overlay.infobox.InfoBoxManager;
 import net.runelite.client.util.ColorUtil;
 import net.runelite.client.util.Text;
 
+import javax.inject.Inject;
```
Comment (deathbeam) [inline runelite-client/src/main/java/net/runelite/client/plugins/clanchat/ClanChatPlugin.java]: https://github.com/runelite/runelite/wiki/Code-Conventions

File: runelite-client/src/main/java/net/runelite/client/plugins/clanchat/ClanChatPlugin.java [hunk 2]
```diff
@@ -173,7 +147,8 @@ public void onClanMemberJoined(ClanMemberJoined event)
 
 			for (final Player player : client.getPlayers())
 			{
-				if (player != null && memberName.equals(Text.toJagexName(player.getName())))
+				if (player != null && memberName.equals(Text.toJagexName(player.getName())) &&
```
Comment (deathbeam) [inline runelite-client/src/main/java/net/runelite/client/plugins/clanchat/ClanChatPlugin.java]: you should instead compare localPlayer name to memberName before loop

Comment (DarkSlaayer) [reply runelite-client/src/main/java/net/runelite/client/plugins/clanchat/ClanChatPlugin.java]: Done!

File: runelite-client/src/main/java/net/runelite/client/plugins/clanchat/ClanChatPlugin.java (hunk 1 above)
Comment (DarkSlaayer) [reply runelite-client/src/main/java/net/runelite/client/plugins/clanchat/ClanChatPlugin.java]: Forgot to disable the auto imports on intellij

Comment (deathbeam) [reply runelite-client/src/main/java/net/runelite/client/plugins/clanchat/ClanChatPlugin.java]: your imports still ahve completely wrong format

**Comment by deathbeam** [comment]
Your imports are still wrong. Just import the codestyle XML from wiki and let intellij format it

**Pull request closed without merging**